src/main/java/animals/localization/LanguagesRulesEn.java

```diff
@@ -0,0 +1,100 @@
+package animals.localization;
+
+import java.util.Map;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class LanguagesRulesEn implements LanguageRule {
+    private static final Pattern STATEMENT = Pattern.compile("it (can|has|is) .+");
+    private static final Scanner SCANNER = new Scanner(System.in);
```

Comment:
This class is responsible for localisation. It means that it should not contain SCANNER and `System.out.printf()`